omod/src/main/webapp/resources/jsonforms/HMIS-TB-001-SecondLineTBTreatmentForm-FollowupSection.json

```diff
@@ -60,7 +60,6 @@
                 {
```

Comment:
Keep name consistent as the HTML form entry form 
https://github.com/METS-Programme/openmrs-module-ugandaemr/blob/6db31049fa3b82eb6b474e3121890cae04c93de6/omod/src/main/webapp/resources/jsonforms/HMIS-TB-001-SecondLineTBTreatmentForm-FollowupSection.json#L2

Reply:
add encounterType  to this form 
https://github.com/METS-Programme/openmrs-module-ugandaemr/blob/6db31049fa3b82eb6b474e3121890cae04c93de6/omod/src/main/webapp/resources/jsonforms/HMIS-TB-001-SecondLineTBTreatmentForm-EnrollmentPage.json#L5

Reply:
Make sure form uuid matches that of the old form "4929f668-f8d0-11ea-adc1-0242ac120002"
https://github.com/METS-Programme/openmrs-module-ugandaemr/blob/6db31049fa3b82eb6b474e3121890cae04c93de6/omod/src/main/webapp/resources/jsonforms/HMIS-TB-001-SecondLineTBTreatmentForm-EnrollmentPage.json#L6C12-L6C48